validation/td-validation.ttl

```diff
@@ -367,15 +367,31 @@ well-known types necessary to implement the WoT interaction model described in
 operations types SHOULD NOT be arbitrarily set by servients
 </span>.
 </p>
+
+<p>
+The optional <code>response</code> name-value pair can be used to provide metadata, such as content type information, that is only itended for the reponse messages. 
+<span class="rfc2119-assertion" id="td-expectedResponse-default-contentType">
+If no <code>response</code> name-value pair is provided it MUST be assumed that the content type of the 
+response is equal to the assigned content type of the form.</span>
+E.g., if the value of the content type of the form is <code>application/xml</code> the assumed value of the content type of the response will be also <code>application/xml</code>.
+</p>
+
+
 <p>
-In some use cases such as for an <code>ActionAffordance</code> there is a different expectation about the content type for the <code>input</code> 
-and <code>output</code>. In such a case, the optional <code>response</code> name-value pair can describe the content type of the expected response.
+In some use cases such as for an <code>ActionAffordance</code> there is a different expectation about the content type 
+for the <code>input</code> and <code>output</code>. In such a case, the optional <code>response</code> name-value pair can describe the content type of the expected response.
 <span class="rfc2119-assertion" id="td-expectedResponse-contentType">
-If the content type of the expected response does not equal its <a>Default Value</a> and differs from the content type of the form,
-the <code>Form</code> instance MUST include a name-value pair with the name <code>response</code>.
+If the content type of the expected response differs from the content type of the form,
+the <code>Form</code> instance MUST include a name-value pair with the name <code>response</code>. 
+E.g., the <code>input</code> of an <code>ActionAffordance</code> accept <code>application/json</code> as content type and its 
```

Comment:
This should be outside of the rfc2119 span.

For instance, ...

Reply:
... An `ActionAffordance` could only accept `application/json` for its input data, while it
will respond with an `image/jpeg` content type for its output data.